app/Http/routes.php

```diff
@@ -12,27 +12,19 @@
 */
 
 
-/*
+/*-------------------------
  * Routes Authentication
+ * ------------------------
  */
 
-Route::group(['middleware' => ['web']], function() {
-    /*
-     * Routes to pages with authentication
-     */
-    Route::auth();
-
-    /*
-    * Routes to the different html pages
-    */
-
-    Route::get('/', 'PagesController@home');
-
-    Route::get('about', 'PagesController@about');
+//Routes to pages with authentication
+Route::auth();
 
-    Route::get('projectfinder', 'PagesController@projectfinder');
+//Routes to the different html pages
+Route::get('/', 'PagesController@home');
+Route::get('about', 'PagesController@about');
+Route::get('projectfinder', 'PagesController@projectfinder');
 
-    //Route::get('register', 'PagesController@register');
-    //Route::get('login', 'PagesController@login');
-    //Route::get('reset', 'PagesContorller@reset');
-});
\ No newline at end of file
+//Route::get('register', 'PagesController@register');
```

Comment:
Oops no it is not. That was the old routing before I used Route::auth(); to control the authentication routing.